scania-digital-design-system/tegel
# feat(button): prefix change

**Describe pull-request**  
Updated prefix for button.

**Solving issue**  
Fixes: [DTS-1628](https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?assignee=63ff6d6c314f5088138187f0&selectedIssue=DTS-1628)

**How to test**  
_Add description how to test if possible_
Go to storybook.
Check in Components -> Button
Check the components APIs and styles.

**Suggested test steps**


[DTS-1628]: https://tegel.atlassian.net/browse/DTS-1628?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
